src/main/java/picard/illumina/CollectIlluminaLaneMetrics.java

```diff
@@ -115,7 +118,7 @@ protected int doWork() {
             }
         }
 
-        IlluminaLaneMetricsCollector.collectLaneMetrics(RUN_DIRECTORY, OUTPUT_DIRECTORY, OUTPUT_PREFIX, laneMetricsFile, phasingMetricsFile, READ_STRUCTURE);
+        IlluminaLaneMetricsCollector.collectLaneMetrics(RUN_DIRECTORY, OUTPUT_DIRECTORY, OUTPUT_PREFIX, laneMetricsFile, phasingMetricsFile, READ_STRUCTURE, FILE_EXTENSION == null ? "" : FILE_EXTENSION);
```

Comment:
Would it make more sense to initialize `FILE_EXTENSION` to `""` instead of `null` and then avoid this null check?  (although I suppose someone could perversely pass `null` in on the command line, but maybe those sorts of people deserver what they get?)   Or is it null to avoid printing a default of `""` on the commandline? )